src/Response/Embeds/Header.php

```diff
@@ -54,6 +54,49 @@ class Header extends AbstractResponse
     public function setDate($date)
     {
         $this->Date = new \DateTime($date);
+
+        return $this;
+    }
+
+    /**
+     * Set Path
+     *
+     * @param string $Path
+     *
+     * @return $this
+     */
+    public function setPath($Path)
+    {
+        $this->Path = $Path;
+
+        return $this;
+    }
+
+    /**
+     * Set ErrorCode
+     *
+     * @param string $ErrorCode
```

Comment:
```suggestion
     * @param string&numeric $ErrorCode
```

This will fix `84: Property Altapay\Response\Embeds\Header::$ErrorCode (string&numeric) does not accept string.`